netmap/node_info.go

```diff
@@ -1,36 +1,23 @@
 package netmap
 
 import (
-	"strconv"
-
-	"github.com/nspcc-dev/hrw"
 	"github.com/nspcc-dev/neofs-api-go/v2/netmap"
 )
 
-type (
-	// Node is a wrapper over NodeInfo.
-	Node struct {
-		ID       uint64
-		Index    int
-		Capacity uint64
-		Price    uint64
-		AttrMap  map[string]string
```

Comment:
The initial idea was to get rid of wrapper type `node` in order to
1. avoid preliminary allocation step with calculating `[]node`
2. avoid returning some type other than `NodeInfo`

I thought than `2.` can be resolved by returning numerical indices `[][]int` instead of `[][]NodeInfo`, but decided to postpone this variant try.

About `1.`: I guess we can't predict real cases (how many attributes, how many filters/selectors, etc.), so approach selection is pretty blind since each variant has pros and cons. Since `NodeInfo` is a "lowest" type for the end user, we can make the `attributes` a `map` within the `NodeInfo` struct itself (`node` type is not needed then). The same goes for subnets